src/main/java/Jarvas.java

```diff
@@ -0,0 +1,104 @@
+import java.util.Scanner;
+public class Jarvas {
+    public static void initialiseBot(){
+        System.out.println(" _____                                  ");
+        System.out.println("(___  )                                 ");
+        System.out.println("    | |   _ _  _ __  _   _    _ _   ___ ");
+        System.out.println(" _  | | /'_` )( '__)( ) ( ) /'_` )/',__)");
+        System.out.println("( )_| |( (_| || |   | \\_/ |( (_| |\\__, \\");
+        System.out.println("`\\___/'`\\__,_)(_)   `\\___/'`\\__,_)(____/");
```

Comment:
Would it be more readable to define this ASCII art as a string constant with escape characters, then call println() on this variable instead?